models/User/AbstractUser.php

```diff
@@ -144,7 +144,7 @@ public function getParentId()
     }
 
     /**
-     * @param int $parentId
+     * @param int|null  $parentId
```

Comment:
I think we should not allow `null` here (in the future) ... thus not adding `null` to the type hints.